src/main/java/racingcar/Application.java

```diff
@@ -1,7 +1,21 @@
 package racingcar;
 
+import java.util.Scanner;
+import java.io.IOException;
+import java.util.Random;
+
 public class Application {
-    public static void main(String[] args) {
-        // TODO: 프로그램 구현
+    public static <string, try1> void main(String[] args) {
+        Scanner in = null;
```

Comment:
스캐너를 이렇게 작성하시면 NPE(NullPointerException)이 발생해요!
[자바 Scanner 사용법](https://st-lab.tistory.com/92)
원하시는 대로 서술하시려면, 아래의 방법처럼 해주셔야 한답니다. 새로운 Scanner 객체를 생성하셔야해요.
```suggestion
        Scanner in = new Scanner(system.in);
```